### PR TITLE
remove the unused Info interface in daemon/execdriver/driver.go

### PR DESCRIPTION
**Pull request by fangyuan930917**
Signed-off-by: Fangyuan Gao 21551127@zju.edu.cn
As #19350 says, Info is not used and its IsRunning interface can be found in container
I just happened to be viewing source code and found @amitkris has not created PR.

### CODE REVIEW

**Comment by fangyuan930917** [comment]
Test failed because read tcp 10.0.0.21:56746->54.164.211.47:443: i/o timeout?

`14:52:48 ----------------------------------------------------------------------
14:52:48 FAIL: docker_cli_events_test.go:182: DockerSuite.TestEventsImagePull
14:52:48 
14:52:48 docker_cli_events_test.go:188:
14:52:48     dockerCmd(c, "pull", "hello-world")
14:52:48 C:/Jenkins/gopath/src/github.com/docker/docker/pkg/integration/dockerCmd_utils.go:35:
14:52:48     c.Assert(err, check.IsNil, check.Commentf("%q failed with errors: %s, %v", strings.Join(args, " "), out, err))
14:52:48 ... value *exec.ExitError = &exec.ExitError{ProcessState:(*os.ProcessState)(0xc082756e20)} ("exit status 1")
14:52:48 ... "pull hello-world" failed with errors: Using default tag: latest
14:52:48 Error response from daemon: Get https://registry-1.docker.io/v2/library/hello-world/manifests/latest: read tcp 10.0.0.21:56746->54.164.211.47:443: i/o timeout
14:52:48 , exit status 1
14:52:48 
14:52:50 `

File: daemon/execdriver/native/info.go [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-type info struct {
-	ID     string
-	driver *Driver
-}
-
-// IsRunning is determined by looking for the
-// pid file for a container.  If the file exists then the
-// container is currently running
-func (i *info) IsRunning() bool {
-	_, ok := i.driver.activeContainers[i.ID]
```
Comment (runcom) [inline daemon/execdriver/native/info.go]: This was also racy afaict

**Comment by calavera** [comment]
LGTM

**Comment by runcom** [comment]
LGTM, could you squash you commits?

**Pull request merged**